packages/genesys-spark-components/src/components/beta/gux-tooltip-beta/gux-tooltip.tsx

```diff
@@ -100,21 +97,18 @@ export class GuxTooltip {
 
   render(): JSX.Element {
     return (
-      <Host id={this.id} role="tooltip">
```

Comment:
I wasn't sure what this was bringing to the table for the component but I may have just missed it. Also, it meant that the ids on the tooltip-beta and the tooltip-base were the same which seemed unusual to me.

Reply:
The `id` is needed as this component will be the one in the light dom